resources/contributors.txt

```diff
@@ -3,80 +3,89 @@
-vanolpfan | https://github.com/vanolpfan |  | Contributor
-23rd | https://github.com/23rd |  | Contributor
-machgo | https://github.com/machgo |  | Contributor
-TranRed | https://github.com/TranRed |  | Contributor
-RAnders00 | https://github.com/RAnders00 | | Contributor
-gempir | https://github.com/gempir | | Contributor
-mfmarlow | https://github.com/mfmarlow | | Contributor
-y0dax | https://github.com/y0dax | | Contributor
-Iulian Onofrei | https://github.com/revolter | :/avatars/revolter.jpg | Contributor
-matthewde | https://github.com/m4tthewde | :/avatars/matthewde.jpg | Contributor
-Karar Al-Remahy | https://github.com/KararTY | :/avatars/kararty.png | Contributor
-Talen | https://github.com/talneoran | | Contributor
-SLCH | https://github.com/SLCH | :/avatars/slch.png | Contributor
-ALazyMeme | https://github.com/alazymeme | :/avatars/alazymeme.png | Contributor
-xHeaveny_ | https://github.com/xHeaveny | :/avatars/xheaveny.png | Contributor
-1xelerate | https://github.com/xel86 | :/avatars/_1xelerate.png | Contributor
-acdvs | https://github.com/acdvs | | Contributor
-karl-police | https://github.com/karl-police | :/avatars/karlpolice.png | Contributor
-brian6932 | https://github.com/brian6932 | :/avatars/brian6932.png | Contributor
-hicupalot | https://github.com/hicupalot | :/avatars/hicupalot.png | Contributor
-iProdigy | https://github.com/iProdigy | :/avatars/iprodigy.png | Contributor
-Jaxkey | https://github.com/Jaxkey | :/avatars/jaxkey.png | Contributor
-Explooosion | https://github.com/Explooosion-code | :/avatars/explooosion_code.png | Contributor
-mohad12211 | https://github.com/mohad12211 | :/avatars/mohad12211.png | Contributor
-Wissididom | https://github.com/Wissididom | :/avatars/wissididom.png | Contributor
-03y | https://github.com/03y | | Contributor
-ScrubN | https://github.com/ScrubN | | Contributor
-Cyclone | https://github.com/PsycloneTM | :/avatars/cyclone.png | Contributor
-2547techno | https://github.com/2547techno | :/avatars/techno.png | Contributor
-ZonianMidian | https://github.com/ZonianMidian | :/avatars/zonianmidian.png | Contributor
-olafyang | https://github.com/olafyang | | Contributor
-chrrs | https://github.com/chrrs | | Contributor
-4rneee | https://github.com/4rneee | | Contributor
-crazysmc | https://github.com/crazysmc | :/avatars/crazysmc.png | Contributor
-SputNikPlop | https://github.com/SputNikPlop | | Contributor
-fraxx | https://github.com/fraxxio | :/avatars/fraxx.png | Contributor
-KleberPF | https://github.com/KleberPF | | Contributor
+fourtf | https://fourtf.com | :/avatars/fourtf.png
+pajlada | https://pajlada.se | :/avatars/pajlada.png
+
+@header Collaborators
+
+zneix | https://github.com/zneix | :/avatars/zneix.png
+Mm2PL | https://github.com/mm2pl | :/avatars/mm2pl.png
+YungLPR | https://github.com/leon-richardt |
+dnsge | https://github.com/dnsge |
+Felanbird | https://github.com/Felanbird |
+kornes | https://github.com/kornes |
+
+@header Contibutors
```

Comment:
This spelling doesn't seem right to me (missing `r`)

Reply:
Oops, thank you for checking!